### PR TITLE
mysql_replication: add basic CI tests for MariaDB from scratch

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
mysql_replication: add basic CI tests for MariaDB from scratch

It needs to complete related PR, for example, https://github.com/ansible/ansible/pull/62648

This tests are **basic** now.
All mysql_replication's features will be covered later.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
```mysql_replication```

### CODE REVIEW

File: test/integration/targets/mariadb_replication/tasks/mariadb_replication_initial.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
```
Comment (gundalow) [inline test/integration/targets/mariadb_replication/tasks/mariadb_replication_initial.yml:1]: Is `mariadb_replication` tested already by this target?

**Comment by Andersson007** [comment]
ready_for_review

File: test/integration/targets/setup_mariadb/tasks/setup_mariadb.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- name: Initialize master's DB
+  shell: 'mysql_install_db --user=mysql --datadir={{ default_datadir }}'
+
+- name: Start services
+  shell: 'mysqld_multi start 1,2'
+
+- name: Pause
+  pause: seconds=3
+
+########### For painful debug uncomment the lines below ##
```
Comment (gundalow) [inline test/integration/targets/setup_mariadb/tasks/setup_mariadb.yml:43]: This is great and will save people time in the future, thanks.

File: test/integration/targets/mariadb_replication/tasks/mariadb_replication_initial.yml (hunk 1 above)
Comment (Andersson007) [reply test/integration/targets/mariadb_replication/tasks/mariadb_replication_initial.yml:1]: no, i made it from scratch (there aren't any ci for mysql_replication now)

Comment (Andersson007) [reply test/integration/targets/mariadb_replication/tasks/mariadb_replication_initial.yml:1]: i added checks for 3 basic options

File: test/integration/targets/setup_mariadb/tasks/setup_mariadb.yml (hunk 2 above)
Comment (Andersson007) [reply test/integration/targets/setup_mariadb/tasks/setup_mariadb.yml:43]: i was suffering for 3 days trying to launch 2 MariaDB instances in one container and configure replication between them :) so decided to left these lines :)

**Review by bmalynovytch — APPROVED** [review]
Good work 👏

**Comment by bmalynovytch** [comment]
shipit

**Comment by Andersson007** [comment]
@bmalynovytch thanks for reviewing and approving!
after that, i hope, we will cope with mysql_replication related PRs (i'll try to rewrite them if authors don't)

**Comment by Andersson007** [comment]
@gundalow , thank you for reviewing and merging!

**Pull request merged**